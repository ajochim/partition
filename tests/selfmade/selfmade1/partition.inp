selfmade.gen
2
36
38
1.2
