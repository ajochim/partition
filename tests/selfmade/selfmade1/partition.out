3
2 19 14
1 2
3 4 5 6 9 10 11 12 13 14 15 16 21 22 23 24 25 26 27
7 8 17 18 19 20 28 29 30 31 32 33 34 35
